app/controllers/hubstats/pull_requests_controller.rb

```diff
@@ -9,8 +9,6 @@ def index
       pull_ids = Hubstats::PullRequest
         .belonging_to_users(params[:users])
         .belonging_to_repos(params[:repos])
-        .state_based_order(@timespan,params[:state],"ASC")
-        .map(&:id)
```

Comment:
The ordering isn't necessary because the labels call below orders what's we get from the pull_ids. The map line was a mistake. I'll fix it.